sections/aws.zsh

```diff
@@ -26,12 +27,12 @@ spaceship_aws() {
   spaceship::exists aws || return
 
   # Is the current profile not the default profile
```

Comment:
I would add it before this line as
```
# Interpolate Native or aws-vualt profile
SPACESHIP_AWS_PROFILE="${AWS_PROFILE:-"$AWS_VAULT"}"
```

You dont want to use `SPACESHIP_AWS_PROFILE=${AWS_PROFILE="$AWS_VAULT"}` as that is basically setting the vault as the `AWS_PROFILE` var, which can cause its own issues and its affecting the environment.
You most likely want `SPACESHIP_AWS_PROFILE=${AWS_PROFILE:-"$AWS_VAULT"}` (notice the `:-`) which makes it so it uses `AWS_PROFILE` unless it's missing, in which case it will use whatever it's after the `:-`, in this case, the vault profile.

Reply:
@hatless See [Parameter-Expansion](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion) for more info.

Reply:
@hatless See [Parameter-Expansion](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion) for more info.